spec/features/authentication_spec.rb

```diff
@@ -0,0 +1,36 @@
+# As a Maker
+# So that only I can post messages on Chitter as me
+# I want to log in to Chitter
+
+feature 'Authentication' do
+  before do
+    User.create(name: 'Joe Bloggs',
+                username: 'jblogg',
+                email: 'joe@bloggs.com',
+                password: 'jamesbond007')
+  end
+
+  it 'A user can sign in' do
```

Comment:
Why have you used it tests in your features spec? obvs not a big deal but for consistency is there a reason you've used different formatting in this features spec file compared to the others?

Reply:
WHOOPS!!!!!!

Reply:
fixed